SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -583,14 +583,19 @@ void USpatialNetDriver::GSMQueryDelegateFunction(const Worker_EntityQueryRespons
 
 	if (!bQueryResponseSuccess)
 	{
-		UE_LOG(LogSpatialOSNetDriver, Error, TEXT("Failed to extract AcceptingPlayers and SessionId from GSM query response. Will retry query for GSM."));
+		UE_LOG(LogSpatialOSNetDriver, Error, TEXT("Failed to extract AcceptingPlayers and SessionId from GSM query response."));
 		RetryQueryGSM();
 		return;
 	}
-	else if (bNewAcceptingPlayers != true ||
-		QuerySessionId != SessionId)
+	else if (!bNewAcceptingPlayers)
 	{
-		UE_LOG(LogSpatialOSNetDriver, Log, TEXT("GlobalStateManager did not match expected state. Will retry query for GSM."));
+		UE_LOG(LogSpatialOSNetDriver, Log, TEXT("GlobalStateManager not accepting players. Did you launch the correct number of servers?"));
```

Comment:
Is "Did you launch the correct number of servers?" a reasonable solution to not being able to accept new players? How does that work?

Reply:
Yeah fair point - it's a little contrived - this bool is only set to true once all the workers have registered themselves and the main gsm server has then acknowledged that the `VirtualWorkerTranslator->IsReady()`. Then it sets accepting players to true. Unfortunately clients don't have any of that info, they just see accepting players is set to true, and the most likely reason this isn't done is the virtual worker translator failure.

Reply:
Maybe worth adding a middle sentence about not all expected workers having declared that they were ready to begin play? then the question about launching correct # of workers makes more sense?